File: C/section03/S03E04.c
#include <stdio.h>

int main() {
    int numberOne, numberTwo, result;

    printf("Please type the number one:\n");
    scanf("%d", &numberOne);

    printf("Please type the number two:\n");
    scanf("%d", &numberTwo);

    result = numberOne + numberTwo;

    printf("\nThe sum is %d\n", result);

    return 0;
}

File: Java/apl-java-exercises/src/br.com.geekuniversity.section03/S03e04.java
package br.com.geekuniversity.section03;

import java.util.Scanner;

public class S03e04 {
    public static void main(String[] args) {
        float numberOne, numberTwo, result;

        Scanner userInput =  new Scanner(System.in);
        
        System.out.println("Please enter the first number: ");
        numberOne = userInput.nextFloat();
        
        System.out.println("Please enter the second number: ");
        numberTwo = userInput.nextFloat();

        result = numberOne + numberTwo;

        System.out.println("\nThe sum result is: " + result);
        
        userInput.close();

    }
}